README.md

```diff
@@ -85,11 +109,15 @@ var obj = {
 
 -- YOUR ANSWER HERE --
 
+- ES5
```

Comment:
Hey Matt, can you double check that this is actually ES6? I looked this up and can't figure out how this is ES6, still looks like ES5 to me. Are we talking about the problem: 
function multiply(x) {
  return x * y;
}